README.md

```diff
@@ -271,6 +271,39 @@ Add to your repository as `.github/workflows/sile.yaml`.
 This work flow assumes your project has a source file `my-document.sil` and will leave behind a `my-document.pdf`.
 Note the comments in [the section about Docker](#docker) regarding version tags.
 
+## Installing third-party packages
+
+Third-party SILE packages rely on the `luarocks` package manager.
+
+To get your Lua version which is used for the execution of `sile`:
```

Comment:
I think we should probably note that for most cases just using the LuaRocks default is probably okay, manually specifying the Lua version only applies when there is both more than one available and SILE was configured to use something other than the default.